### DISCUSSION TITLE
How create increment column in visidata from a set value, not for 1?

### DISCUSSION THREAD

**Discussion by Freed-Wu** [saulpw/visidata#1788] [Q&A] [answered]
<https://superuser.com/questions/1722292/how-create-increment-column-in-visidata-from-a-set-value-not-for-1>

**Comment by saulpw** [comment, marked as answer]
`options.incr_base`

**Reply by Freed-Wu** [reply]
How can we define a hotkey like `zgi <number1> <number2>`, will create a column which value is `number1, number1 + number2, number1 + number2 * 2, ...`?
